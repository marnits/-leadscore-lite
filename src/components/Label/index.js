import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles.module.scss';

const Label = ({ htmlFor, text }) => (
  <label htmlFor={htmlFor} className={styles.Label}>{text}</label>
);

Label.defaultProps = {
  htmlFor: null,
};

Label.propTypes = {
  htmlFor: PropTypes.string,
  text: PropTypes.string.isRequired,
};

export default Label;
