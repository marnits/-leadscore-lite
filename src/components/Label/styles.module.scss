.Label {
  font-size: 12px;
  color: #545454;
  font-weight: bold;
  padding: 6px 13px;
}
