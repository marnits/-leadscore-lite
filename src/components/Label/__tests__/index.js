import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../index';

describe('Label', () => {
  test('should render component properly', () => {
    const tree = renderer.create(<Label htmlFor="test" text="label" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
