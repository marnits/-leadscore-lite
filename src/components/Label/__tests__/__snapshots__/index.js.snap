// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render component properly 1`] = `
<label
  className="Label"
  htmlFor="test"
>
  label
</label>
`;
